tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "tslint-config-standard",
+  "rules": {
+    "no-floating-promises": false
```

Comment:
may want to grab this from citizen frontend, we also stop console.log from being committed

Reply:
Just done that, going to push it shortly 👍